bot/exts/evergreen/trivia_quiz/_game.py

```diff
@@ -0,0 +1,138 @@
+
+            await self.ctx.send(embed=start_embed)  # send an embed with the rules
+            await asyncio.sleep(1)
+
+        # A variable to store the questions which have been sent to the user during the quiz.
+        done_question = []
+        # A variable to store the number of hints used in a quiz question.
+        hint_no = 0
+        # A variable to store the answer to a quiz question.
+        answer = None
+
+        while not self.over:
+            # Exit quiz if number of questions for a round are already sent.
+            if len(done_question) > _constants.QUESTION_LIMIT and hint_no == 0:
+                await self.ctx.send("The round has ended.")
+                await _helpers.declare_winner(self.ctx.channel, self.player_scores)
+                self.over = True
+                break
+
+            # If no hint has been sent or any time alert. Basically if hint_no = 0  means it is a new question.
+            if hint_no == 0:
+                # Select a random question which has not been used yet.
+                while True:
+                    question_dict = random.choice(self.questions)
+                    if question_dict["id"] not in done_question:
+                        done_question.append(question_dict["id"])
+                        break
+
+                q = question_dict["question"]
+                answer = question_dict["answer"]
+
+                embed = discord.Embed(
+                    title=f"Question #{len(done_question)}",
+                    description=q,
+                    colour=discord.Colour.gold()
+                )
+                await self.ctx.send(embed=embed)  # Send question embed.
+
+            # A function to check whether user input is the correct answer(close to the right answer)
+            def check(m: discord.Message) -> bool:
+                return (
+                    m.channel == self.ctx.channel
+                    and fuzz.ratio(answer.lower(), m.content.lower()) > _constants.CORRECTION_FUZZ_RATIO
+                )
+
+            try:
+                msg = await self.bot.wait_for('message', check=check, timeout=_constants.MESSAGE_TIMEOUT)
+            except asyncio.TimeoutError:
+                # In case of TimeoutError and the game has been stopped, then do nothing.
+                if self.over:
+                    break
+
+                # if number of hints sent or time alerts sent is less than 2, then send one.
+                if hint_no < _constants.HINTS_LIMIT:
+                    hint_no += 1
+                    if "hints" in question_dict:
+                        hints = question_dict["hints"]
+                        await self.ctx.send(f"**Hint #{hint_no + 1}\n**{hints[hint_no]}")
+                    else:
+                        # If hints are not available for the question, then send time alerts
+                        await self.ctx.send(
+                            f"{(_constants.HINTS_LIMIT * 10) - (hint_no * 10)}s left!"
```

Comment:
This line is causing this issue, with wrong time left being sent.
![image](https://user-images.githubusercontent.com/37217450/118378206-b8f7f080-b5ef-11eb-8d72-f8e0a2577c5f.png)


Solution is to increment the `hint_no` value after sending the hint or the time (after line 97).

Reply:
Also in constants, the `MESSAGE_TIMEOUT` is 15 seconds, but the time left is calculated by taking the value 10 seconds.